### PR TITLE
xcode-select: add page

### PR DESCRIPTION
**Pull request by NSExceptional**
Add page for xcode-select

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by schneiderl** [comment]
We might need a little help with Travis here @sbrl @owenvoke.

**Comment by owenvoke** [comment]
Interesting, I've had a look and Travis seems to say that it detected abuse (but no information why).

They say the following:
>... if you believe you have been flagged wrongly, the best thing to do is to contact us at support@travis-ci.com with details (repository, PR URL, etc.).

**Comment by zdroid** [comment]
I sent the mail to that email address. Hopefully this will be resolved. Sorry @NSExceptional for all the trouble.

**Comment by NSExceptional** [comment]
Thanks! All good, this is wild

**Comment by zdroid** [comment]
@NSExceptional Your account was wrongly flagged for abuse. It's been cleared by Travis support, so this is ready for merging now.

**Comment by schneiderl** [comment]
Nice, thank you for contacting Travis Support @zdroid

File: pages/osx/xcode-select.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+> Switch between different versions of Xcode and the included developer tools.
+> Also used to update the path to Xcode if it is moved after installation.
+
+- Install Xcode's command-line tools:
+
+`xcode-select --install`
+
+- Select a given path as the active developer directory:
+
+`xcode-select -s {{path/to/Xcode.app/Contents/Developer}`
```
Comment (owenvoke) [inline pages/osx/xcode-select.md]:
```suggestion
`xcode-select -s {{path/to/Xcode.app/Contents/Developer}}`
```

**Comment by NSExceptional** [comment]
It says one change requested but I committed the requested change?

**Comment by NSExceptional** [comment]
Haha thanks. I bet I'm flagged for spam again or something. I'll keep this in mind next time — I plan on contributing a lot!

I would still appreciate it if someone could reach back out to Travis and see what happened. If there's something I can do differently to avoid this. Unless this is a normal occurrence

**Comment by sbrl** [comment]
> I plan on contributing a lot

That'd be really cool :D

I'm sure it's just a one-off.

**Pull request merged**